FAQ.md

```diff
@@ -71,4 +71,37 @@ Just add an image property to the page frontmatter. For example:
 image: "/img/episode_preview_1200x630px.png"
 ```
 
+## How do I add a new script to the page?
+
+Add the script to the `_scripts` folder, then be sure to add it to the entry section
+on the webpack configuration (`webpack.config.js`) like so:
+
+```
+entry: {
+  // ... other scripts
+  'new-script': './_scripts/new-script.js'
+},
+```
+
+Then if you are already running `yarn watch`, cancel it and run it again.
+
+## Why do I need to run `yarn serve` and `yarn watch` at the same time?
```

Comment:
@TheMightyPenguin please clarify why we can not do the same with jekyll like a brief support of why we need the wbepack